packages/@atjson/source-commonmark/src/source.ts

```diff
@@ -57,7 +66,7 @@ export default class CommonMarkSource extends Document {
   }
 
   static get markdownParser() {
-    return MarkdownIt("commonmark");
+    return new MarkdownIt("commonmark", { linkify: false });
```

Comment:
Any reason we're moving this around?

Reply:
it was originally so that we could pass parser arguments into `fromRaw` but I think I decided that wasn't necessary at some point lol...I can revert this just to reduce the surface area of this PR